libs/common-services/src/test/java/uk/gov/di/ipv/core/library/fixtures/VcFixtures.java

```diff
@@ -280,93 +290,115 @@ public interface VcFixtures {
                     "personalNumber", "MORGA753116SM9IJ",
                     "expiryDate", "2042-10-01");
 
-    String PASSPORT_NON_DCMAW_SUCCESSFUL_VC =
+    VerifiableCredential PASSPORT_NON_DCMAW_SUCCESSFUL_VC =
             generateVerifiableCredential(
+                    "urn:uuid:e6e2e324-5b66-4ad6-8338-83f9f837e345",
```

Comment:
What is this magic string? Should it be a constant, or handled internally to `generateVerifiableCredential`?

Reply:
Good point - I'll make it a constant defined at the top